### ISSUE TITLE
Cannot convert undefined or null to object after publishing a page

### ISSUE THREAD

**Issue by robbieaverill** [dnadesign/silverstripe-elemental#542]
**Version: SS 4.3.x-dev and elemental 4.0.x-dev**

* Create a page
* Add a content block
* Publish the page

When the editor reloads, notice a console error:

```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at entwine.js:61
```

It doesn't seem to break the functionality though.

It comes from entwine.js:

```js
          // We can introspect the store to find form names in the `element` namespace
          store.dispatch(destroy(
            ...Object.keys(store.getState().form.formState.element).map(name => `element.${name}`)
          ));
        }, 0);
```

If you re-publish the page again you also get this error:

```
vendor.js?m=1543760163:1 Uncaught (in promise) Error: Network error: Store reset while query was in flight(not completed in link chain)
```

**Comment by Leapfrognz** [comment]
Experiencing the same bug. SS 4.3.0 CWP 2.2

**Comment by clintLandrum** [comment]
Experiencing the same bug but this **does** break functionality for me.

Any TinyMCE (HTMLEditorField) formatting/saving doesn't work which prevents any edits to these fields from being made. If others are experiencing this too it might justify updating this issue with a label of **impact/high**

**Comment by robbieaverill** [comment]
Fixed in elemental 4.1 with #691
